src/services/jwtServices.js

```
@@ -0,0 +1,32 @@
+import jwt from 'jsonwebtoken';
+
+function sign(user) {
+  return jwt.sign(user, process.env.JWT_KEY, { expiresIn: '1h' });
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
```

let's throw an error instead of returning null